pkg/api/v1/atlasbackupcompliancepolicy_types_test.go

```diff
@@ -0,0 +1,130 @@
+package v1
```

Comment:
These tests would move to conversion unit test within the translation layer, once we go there.